datasets/jigsaw_toxicity_pred/jigsaw_toxicity_pred.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow datasets Authors and the HuggingFace, DataLab Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+import os
+
+import pandas as pd
+
+# import datasets
+
+import datalabs
+from datalabs import get_task, TaskType
+
+_DESCRIPTION = """\
+This dataset consists of a large number of Wikipedia comments which have been labeled by human raters for toxic behavior.
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/data"
+
+
+_URL = "https://s3.amazonaws.com/datalab-hub/toxicity_detection/jigsaw_toxicity_pred.zip"
+
+
+_LICENSE = 'The "Toxic Comment Classification" dataset is released under CC0, with the underlying comment text being governed by Wikipedia\'s CC-SA-3.0.'
+
+
+class JigsawToxicityPred(datalabs.GeneratorBasedBuilder):
+    """This is a dataset of comments from Wikipedia’s talk page edits which have been labeled by human raters for toxic behavior."""
+
+    VERSION = datalabs.Version("1.1.0")
+
+    def _info(self):
+
+        return datalabs.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datalabs.Features(
```

Comment:
do we want to define multiple sub-datasets for this dataset? For example,
load_dataset("jigsaw_toicity_pred", "thread")